src/renderer/util.ts

```diff
@@ -19,6 +19,32 @@ export const extractFileExtension: (filePath: string) => string | null = (
   return extension;
 };
 
+export const extractFileNameWithExtension: (
```

Comment:
I think this should be a backend function, that way the renderer doesn't need to care about OS stuff. We might need a getOS() down the line but we're trying to make the platforms have feature parity so I can't foresee anything specific

Reply:
enum refactor here: https://github.com/patrickbrett/mlvet/pull/38/commits/f54cbe3641987ce328de2f5e4091eb81a6eb50d4

I chose to keep getOS since we might need it down the line